handbook/demand/README.md

```diff
@@ -51,12 +51,19 @@ The Head of Demand is the DRI for deploying Fleet's event budget, and events are
   - who from Fleet will attend
   - the training plan for Fleet attendees
   - which talk proposal Fleet will submit
-  - all event materials, including printouts, banners, swag given out, and even the clothing worn by fleeties
-  - estimated budget, including sponsorship or airfare, and lodging for attendees
+  - all event materials, including printouts, banners, swag given out, and even the clothing worn by Fleeties
+  - estimated budget, including sponsorship or airfare and lodging for attendees
```

Comment:
```suggestion
  - estimated budget, including sponsorship or airfare, and lodging for attendees
```
Beleive it or not we want that coma there. https://fleetdm.com/handbook/company/communications#commas